third_party/packages/flutter_svg/test/widget_svg_test.dart

```diff
@@ -39,6 +39,28 @@ Future<void> _checkWidgetAndGolden(Key key, String filename) async {
   await expectLater(widgetFinder, matchesGoldenFile('golden_widget/$filename'));
 }
 
+class _TestColorMapper extends ColorMapper {
+  const _TestColorMapper();
+
+  @override
+  Color substitute(
```

Comment:
Can you please add a doc string that explains the intent of this function?

Reply:
Just a one liner, it helps explain the intent of this test helper to someone reading the files. The current comments in this test are perplexing at first glance.

Reply:
Done!